doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md

```diff
@@ -2442,15 +2442,15 @@ where
 
 *q<sub>oncycloss,n</sub>* and *q<sub>offcycloss,n</sub>* are defined as:
 
-<div>$${q_{oncycloss,n}} = U{A_{oncyc,n}}({T_{amb}} - {T_n})$$</div>
+<div>$${q_{oncycloss,n}} = {UA_{tank}}({T_{amb}} - {T_n})$$</div>
```

Comment:
@nmerket If iam following this correclyt, it seems like UA_tank should either have ",n" or another term for UA_node,n to show this could vary by node due to Tank.AdditionalLossCoeff( NodeNum )